src/main/groovy/com/ullink/gradle/nunit/NUnit.groovy

```diff
@@ -19,24 +22,28 @@ class NUnit extends ConventionTask {
     def verbosity
     def config
     def timeout
+
+    // TODO: shouldn't we just make it a single variable and detect which case it is when invoking NUnit instead?
```

Comment:
>// TODO: shouldn't we just make it a single variable and detect which case it is when invoking NUnit instead?

runList and run are the same as testList and test? Only that one is for nunit3 and the other for nunit2?